Shopist/ShopistUITests/ShopistUITests.swift

```diff
@@ -1,18 +1,57 @@
 /*
-* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
-* This product includes software developed at Datadog (https://www.datadoghq.com/).
-* Copyright 2019-2020 Datadog, Inc.
-*/
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
 
 import XCTest
 
 class ShopistUITests: XCTestCase {
-    override func setUp() {
+    override func setUp() { // swiftlint:disable:this empty_xctest_method
         continueAfterFailure = false
     }
 
-    func testAppCanBeLaunched() {
+    func testRunFullCheckoutFlow() {
+        let sessionCount = 3
         let app = XCUIApplication()
-        app.launch()
+        for sessionIndex in 0...sessionCount {
+            app.launch()
+
+            app.collectionViews.firstMatch.goToOffScreenCell("lastCell")?.tap()
+            app.collectionViews.firstMatch.goToOffScreenCell("lastCell")?.tap()
+
+            let navBar = app.navigationBars.firstMatch
+            navBar.buttons["cart.badge.plus"].tap()
+            navBar.buttons["cart"].tap()
+
+            let cardButton = app.navigationBars["Cart"].buttons["creditcard"]
+            cardButton.tap()
+            if !app.alerts.firstMatch.waitForExistence(timeout: 5.0) {
+                fatalError("Alert should show up!")
```

Comment:
If it crashes for, let's say 3th iteration out of 20, we will lost data for 17 iterations, leaving the gap in time series. We shouldn't crash the tests.

Reply:
as the demo data must be consistent all the time, if anything goes wrong shouldn't we discard the run? a previous batch will be used at worst case. if we continue after failure, inconsistent data will be sent to demo org

Reply:
Data consistency will be ensured with the help of `shopist_org2` workflow run on the `master` or feature branch. What I mean here is inconsistency due to tests flakiness, i.e. slow CI run exceeding this `5.0s` timeout.

Reply:
there should not be that much flakiness in those scenarios, if it happens then there is probably an actual problem rather than just flakiness

let's say Shopist backend is down for a moment, do we prefer no new data or wrong data in RUM Explorer?
if wrong data is not a problem then `shopist_org2` workflow is only nice-to-have and not needed.
if wrong data is a problem then we shouldn't send data from unexpected scenarios.